### ISSUE TITLE
On Windows, CMake gives error with gfortran.

### ISSUE THREAD

**Issue by chrisdembia** [ampl/coin#11]
Hello. I'm hoping to use your CMake files to build Ipopt on Windows. My understanding is that the C++ compiler should be Visual Studio 2015 and the Fortran compiler should be gfortran from MinGW. I tried to use these compilers and CMake generated the following error (could not use gfortran):

```
Checking whether the Fortran compiler is Compaq using "-what" did not match "Compaq Visual Fortran":
gfortran.exe: error: unrecognized command line option '-what'; did you mean '-gnat'?
gfortran.exe: fatal error: no input files
compilation terminated.
Checking whether the Fortran compiler is NAG using "-V" did not match "NAG Fortran Compiler":
gfortran.exe: error: unrecognized command line option '-V'
gfortran.exe: fatal error: no input files
compilation terminated.
Checking whether the Fortran compiler is Compaq using "-what" did not match "Compaq Visual Fortran":
gfortran.exe: error: unrecognized command line option '-what'; did you mean '-gnat'?
gfortran.exe: fatal error: no input files
compilation terminated.
Checking whether the Fortran compiler is NAG using "-V" did not match "NAG Fortran Compiler":
gfortran.exe: error: unrecognized command line option '-V'
gfortran.exe: fatal error: no input files
compilation terminated.
Determining if the Fortran compiler works failed with the following output:
Change Dir: C:/Users/chris/repos/amplcoin/tryfortran/build/CMakeFiles/CMakeTmp

Run Build Command:"C:/Program Files (x86)/Microsoft Visual Studio 14.0/Common7/IDE/devenv.com" "CMAKE_TRY_COMPILE.sln" "/build" "Debug" "/project" "cmTC_122b1"


Microsoft Visual Studio 2015 Version 14.0.25420.1.

Copyright (C) Microsoft Corp. All rights reserved.



Some errors occurred during migration. For more information, see the migration report:

C:\Users\chris\repos\amplcoin\tryfortran\build\CMakeFiles\CMakeTmp\UpgradeLog.htm



The operation could not be completed. The parameter is incorrect. 
```

I have the same issue when trying to use gfortran in a separate trivial CMake project:

```cmake
cmake_minimum_required(VERSION 3.10)
project(foo Fortran)
enable_language(Fortran)
```

So this is not a bug with your project. But it sounds like you've gotten this to work before, so I am wondering what I should do differently.

Any help you could provide would be appreciated. Thank you.

**Comment by fdabrandao** [comment]
The issues with gfortran were addressed in 690ede9113e0fbe846a6d18f2a251aab61a749a1. The build steps using the combination of Visual Studio with gfortran from MinGW are included in the Azure Pipelines configuration at https://github.com/ampl/coin/blob/master/azure-pipelines.yml.

**Comment by chrisdembia** [comment]
Woohoo! Thank you so much.
